frontend/src/routes/manage/Realm/Content/Edit/EditMode/Series.tsx

```diff
@@ -107,53 +92,54 @@ export const EditSeriesBlock: React.FC<EditSeriesBlockProps> = ({ block: blockRe
             onChange={data => seriesField.onChange(data?.id)}
             onBlur={seriesField.onBlur}
         />
-
-        <Heading>{t("series.settings.order")}</Heading>
-        <NiceRadio breakpoint={0}>
-            <NiceRadioOption
-                value="NEW_TO_OLD"
-                defaultChecked={order === "NEW_TO_OLD"}
-                {...form.register("order")}
-            >{t("series.settings.new-to-old")}</NiceRadioOption>
-            <NiceRadioOption
-                value="OLD_TO_NEW"
-                defaultChecked={order === "OLD_TO_NEW"}
-                {...form.register("order")}
-            >{t("series.settings.old-to-new")}</NiceRadioOption>
-        </NiceRadio>
-
-        <Heading>{t("manage.realm.content.series.layout.heading")}</Heading>
-        <LayoutChooser {...{ currentLayout, form }} />
+        <div
+            role="group"
+            aria-labelledby={t("manage.realm.content.display-options")}
```

Comment:
Could it be better to have a group for each DisplayOptionGroup? I tried it with the Fedora screen reader, which added context about the selected group and it read `Panel Reihenfolge` instead of `h3 Reihenfolge`. Or is that too verbose, and display options is enough?

`aria-labelledby` should be replaced with `aria-label`:
```suggestion
            aria-label={t("manage.realm.content.display-options")}
```

Reply:
Oh yeah, right again. Sorry for missing this, we have a lot going on right now. Would you be open to adjust that accordingly in https://github.com/elan-ev/tobira/pull/1058?
Regarding groups for each display option, I'm not sure what works best for screenreader users.
I'm testing with VoiceOver on macOS, which indicates that I am entering and exiting the group, and for the individual panels only reads their heading. I feel like that is enough? But I wouldn't mind trying what you suggested - could you maybe include that in your PR as well?